tests/functional_tests/models/qwen_omni/test_qwen3_omni_conversion.py

```
@@ -0,0 +1,217 @@
+            str(output_dir),
+            "--tp",
+            "2",
+            "--pp",
+            "1",
+            "--ep",
+            "1",
+            "--etp",
+            "1",
+        ]
+
+        result = subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)